pkg/cmd/pulumi/new.go

```diff
@@ -843,6 +1114,35 @@ func pythonCommands() []string {
 	return commands
 }
 
+// Prompt the user to decide whether they'd like to enter an interactive AI prompt or use a template.
+func chooseWithAIOrTemplate(opts display.Options) (string, error) {
+	// Customize the prompt a little bit (and disable color since it doesn't match our scheme).
+	surveycore.DisableColor = true
+
+	options := []string{
+		"ai",
+		"template",
+	}
+
+	optionsDescriptionMap := map[string]string{
+		"ai":       "Create a new Pulumi project using Pulumi AI",
```

Comment:
I expect we should default to the normal template for now, instead of defaulting to the AI option.  As a result I also think we should invert the phrasing of the prompt (or make it more neutral) like "Would you like to create a project from a template or using a Pulumi AI prompt?".

I still do wonder if there is some way for the initial prompt to go straight into the template list if you type anything (like `a` for `aws-typescript`), but stay in the AI vs. template selector if you press up/down/enter.

Reply:
Interesting - the feedback we got during the design phase was to default to AI (put it on top). I can certainly reverse the order, though.

Reply:
> I still do wonder if there is some way for the initial prompt to go straight into the template list if you type anything (like a for aws-typescript), but stay in the AI vs. template selector if you press up/down/enter.

I did look into this as I was starting work here - it seems to me that it's impossible when using `Survey`. I would _love_ to be wrong, though.